src/test/java/redis/clients/jedis/tests/ShardedJedisPoolWithCompleteCredentialsTest.java

```diff
@@ -145,49 +146,6 @@ public void checkFailedJedisServer() {
     pool.destroy();
   }
 
-  @Test
-  public void shouldReturnActiveShardsWhenOneGoesOffline() {
```

Comment:
@sazzad16 why did your remove this test?

Reply:
@gkorland https://github.com/redis/jedis/pull/2368#issuecomment-790262400